bandit/core/blacklisting.py

```diff
@@ -7,13 +7,15 @@
 import ast
 import fnmatch
 
+from bandit.core.cwemap import CWEMAP
 from bandit.core import issue
 
 
 def report_issue(check, name):
     return issue.Issue(
         severity=check.get('level', 'MEDIUM'), confidence='HIGH',
         text=check['message'].replace('{name}', name),
+        cwe=CWEMAP[check.get("id", 'LEGACY')],
```

Comment:
NOTSET or UNDEF?

Reply:
Yes indeed `NOTSET` may be more clear. Just renamed the constant from `UNDEF` to `NOTSET`.

Reply:
Can we rename LEGACY to NOTSET as well?

Reply:
> Can we rename LEGACY to NOTSET as well?

Mhh, I suppose that this would be a question for @dvyakimov and/or @ericwb as the `LEGACY` key was not introduced by the current PR and I am not 100% sure if this change could have any undesirable side-effects.

Reply:
I think LEGACY is fine for now.